torchgeo/datamodules/chesapeake.py

```diff
@@ -112,7 +113,7 @@ def __init__(
         self.test_splits = test_splits
         self.class_set = class_set
         self.use_prior_labels = use_prior_labels
-        self.prior_smoothing_constant = prior_smoothing_constant
+        self.prior_smoothing_constant = torch.tensor(prior_smoothing_constant)
```

Comment:
Same bug as https://github.com/pytorch/pytorch/issues/116328, mypy doesn't like us adding a float to a Tensor anymore.